server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -259,34 +269,103 @@ private TrTag renderTIRow(AccountModel ti) {
+    }
+    ApplicantData applicantData = newestApplicant.get().getApplicantData();
+    Optional<String> maybePhoneNumber = applicantData.getPhoneNumber();
+    String email = account.getEmailAddress();
+
+    return div(
+        label("Contact information").withFor("card_contact_info").withClass("whitespace-nowrap"),
+        div()
+            .condWith(
+                maybePhoneNumber.isPresent(),
+                div(
+                        Icons.svg(Icons.PHONE).withClasses("h-3", "w-3", "mr-1"),
+                        p(formatPhone(maybePhoneNumber.orElse(""))))
+                    .withClass("flex items-center"))
+            .condWith(
+                email != null && !email.isEmpty(),
+                div(Icons.svg(Icons.EMAIL).withClasses("h-3", "w-3", "mr-1"), p(email))
+                    .withClass("flex items-center"))
+            .withClass("text-xs")
+            .withId("card_contact_info"));
+  }
+
+  private String formatPhone(String phone) {
+    return phone.replaceFirst("(\\d{3})(\\d{3})(\\d+)", "($1) $2-$3");
+  }
+
+  private DivTag renderCardApplications(AccountModel account) {
+    Optional<ApplicantModel> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return div();
+    }
+    int applicationCount = newestApplicant.get().getApplications().size();
+
+    String programs =
+        newestApplicant.get().getApplications().stream()
+            .map(
+                application ->
+                    programRepository.getProgramDefinition(application.getProgram()).adminName())
+            .collect(Collectors.joining(", "));
+
+    return div(
+        label(
+                String.format(
+                    "%s application%s submitted",
+                    applicationCount, applicationCount == 1 ? "" : "s"))
+            .withFor("card_applications")
+            .withClass("whitespace-nowrap"),
+        p(programs).withClass("text-xs").withId("card_applications"));
   }
 
-  private TdTag renderUpdateClientInfoCell(Long accountId) {
-    return td().with(
-            new LinkElement()
-                .setId("edit-client")
-                .setText("Edit")
-                .setHref(
-                    controllers.ti.routes.TrustedIntermediaryController.editClient(accountId).url())
-                .asAnchorText())
-        .withClasses(BaseStyles.TABLE_CELL_STYLES, "pr-12");
+  private DivTag renderCardNotes(String notes) {
+    return div(
+        label("Notes").withFor("card_notes"), p(notes).withClass("text-xs").withId("card_notes"));
+  }
+
+  private ATag renderUpdateClientLink(Long accountId) {
+    return new LinkElement()
+        .setId("edit-client")
+        .setText("Edit")
+        .setHref(controllers.ti.routes.TrustedIntermediaryController.editClient(accountId).url())
+        .asAnchorText();
   }
 
-  private TdTag renderDateOfBirthCell(AccountModel account) {
+  private DivTag renderCardDateOfBirth(AccountModel account) {
```

Comment:
Rename to RenderClientDateOfBirth may be?

Reply:
Sure, will do that.